server/belga/search_providers.py

```diff
@@ -212,6 +212,90 @@ def format_list_item(self, data):
         }
 
 
+class Belga360ArchiveSearchProvider(superdesk.SearchProvider):
+
+    GUID_PREFIX = 'urn:belga.be:360archive:'
+
+    label = 'Belga 360 Archive'
+    base_url = 'http://mules.staging.belga.be:48080/belga360-ws/'
+    search_endpoint = 'archivenewsobjects'
+    items_field = 'newsObjects'
+    count_field = 'nrNewsObjects'
+
+    def __init__(self, provider):
+        super().__init__(provider)
+        self.session = requests.Session()
+
+    def url(self, resource):
+        return urljoin(self.base_url, resource.lstrip('/'))
+
+    def find(self, query, params=None):
+        api_params = {
+            'start': query.get('from', 0),
+            'pageSize': query.get('size', 25),
+        }
+
+        try:
+            api_params['searchText'] = query['query']['filtered']['query']['query_string']['query']
+        except KeyError:
+            api_params['searchText'] = ''
+
+        data = self.api_get(self.search_endpoint, api_params)
+        docs = [self.format_list_item(item) for item in data[self.items_field]]
+        return BelgaListCursor(docs, data[self.count_field])
+
+    def fetch(self, guid):
+        _id = guid.replace(self.GUID_PREFIX, '')
+        data = self.api_get(self.search_endpoint + '/' + _id, {})
+        return self.format_list_item(data)
+
+    def api_get(self, endpoint, params):
+        url = requests.Request('GET', 'http://example.com/' + endpoint, params=params).prepare().path_url
```

Comment:
what is this doing?

Reply:
It uses for testing search parameters, this part should be `self.url(endpoint)`, not `example.com`